client/components/vertical-nav/item/index.jsx

```diff
@@ -67,8 +68,14 @@ class VerticalNavItem extends Component {
 
 		const linkProps = external ? { target: '_blank', rel: 'noreferrer' } : {};
 
+		const pathObj = disabled
```

Comment:
I wanted to use a better approach here.

Instead of having a path object and spreading it, I wanted to do something like:

```suggestion
		const WrapperElem = disabled ? 'span' : 'a';
		
		return <WrapperElem href={ path } onClick={ onClick } { ...linkProps }>
```
Benefits include:
1. No need to disable any eslint errors
2. The `href` attribute would automatically be ignored.

I had to decide against it because some folks might use something like:
```CSS
.vertical-nav a { /* some CSS */ }
``` 

Yay or nay?